debian/control

```diff
@@ -2,22 +2,56 @@ Source: intelmq
 Maintainer: Sebastian Wagner <wagner@cert.at>
 Section: python
 Priority: optional
-Build-Depends: debhelper (>= 4.1.16), python3-all, python3-setuptools, quilt, dh-python, python-setuptools, dh-systemd, safe-rm, python3-requests, python3-redis, python3-dnspython, python3-psutil, python3-dateutil, python3-termstyle, python3-tz, lsb-release
+Build-Depends: debhelper-compat (= 10),
```

Comment:
That doesn't work with Debian 9.0 and Ubuntu 16.04 now. If we gonna drop support for them, as they use python 3.5, that would be fine.

Reply:
Oke, reverted to `9`